ci/docker-compose.yml

```diff
@@ -45,7 +45,6 @@ services:
       CONJUR_APPLIANCE_URL: http://conjur
       CONJUR_ACCOUNT: cucumber
       DATABASE_URL: postgres://postgres@pg/postgres
-      AUDIT_DATABASE_URL:
```

Comment:
this commit can be squashed into the previous one. it looks like a leftover.

Reply:
This commit remove the AUDIT_DATABASE_URL env variable which is not in used.
While the previous commit refactor AUDIT_DATABASE_URL to DATABASE_URL